pkg/reconciler/route/controller.go

```diff
@@ -93,6 +93,7 @@ func newControllerWithClock(
 		configStore.WatchConfigs(cmw)
 		return controller.Options{ConfigStore: configStore}
 	})
+	c.enqueueAfter = impl.EnqueueAfter
```

Comment:
Generally we try to avoid having the Reconciler know anything about queueing 😞

Reply:
We do that in KPA as well :/
How else to schedule something in future?